test_scripts/API/SendLocation/001_SendLocation_success.lua

```diff
@@ -0,0 +1,103 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/3
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/SetInteriorVehicleData.md
+-- Item: Use Case 1: Main Flow
+--
+-- Requirement summary:
+-- [SDL_RC] Set available control module settings SetInteriorVehicleData
+--
+-- Description:
+-- In case:
+-- 1) Application is registered with REMOTE_CONTROL appHMIType
+-- 2) and sends valid SetInteriorVehicleData RPC with valid parameters
+-- SDL must:
+-- 1) Transfer this request to HMI
+-- 2) Respond with <result_code> received from HMI
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common_send_location = require('test_scripts/API/SendLocation/common_send_location')
```

Comment:
@Itileda I propose to use module name variable in camelCase:
`local commonSendLocation = require('test_scripts/API/SendLocation/common_send_location')` 
or
`local commonSL = require('test_scripts/API/SendLocation/common_send_location')`

Reply:
@aderiabin , fixed in 112b3f8525256dc4707265b8877dceb952e2fa92